src/chapters/chapter_1/induction.tex

```diff
@@ -3,8 +3,7 @@ \subsection{Prove that $\sum_{i=1}^{n} i = n(n+1)/2$ for $n \geq 0$, by inductio
 
   \begin{proof}[Proof by induction]
 
-    For any integer value of $i$, let $P(n)$ denote the statement $P(n) =
-    n(n+1)/2$.
+    For any integer value $n > 0$, let $P(n)$ denote the statement $P(n) = (\sum_{i=1}^n i \doteq n(n+1)/2)$.
```

Comment:
For all n, P(n) must be a proposition, i.e. evaluate to a truth value. You previously defined P(n) to be a number for every n.